src/status_im/ui/screens/discover/styles.cljs

```diff
@@ -61,37 +70,55 @@
    :padding-right  9})
 
 (def separator
-  {:background-color "rgb(200, 199, 204)"
-   :height           0.5})
+  {:background-color "rgb(238, 241, 245)"
+   :height           2
+   :margin-top       2
+   :margin-bottom    2})
 
 ;; Popular list item
 
 (def popular-list-container
   {:flex             1
    :background-color :white
    :padding-top      18
-   :padding-left     16})
+   :padding-left     16
+   })
 
 (def popular-list-item
-  {:flex-direction :row
+  {:flex-direction "column"
    :padding-bottom 16
+   :margin-right   10
    :top            1})
 
-(def popular-list-item-name
-  {:color          "black"
-   :font-size      15
-   :padding-bottom 4})
+(def popular-list-item-second-row
+  {:flex            1
+   :flex-direction  :row
+   :align-items     :center
+   :justify-content :space-between
+   :margin-bottom   5
+   :padding-top     25})
 
 (def popular-list-item-name-container
-  {:flex           0.8
-   :flex-direction "column"
-   :padding-top    16})
+  {:flex            0.3
+   :flex-direction  :row
+   :justify-content :flex-start})
+
+(def popular-list-item-name
+  {:margin-left 7
+   :color       "black"
+   :font-size   12})
 
 (def popular-list-item-avatar-container
-  {:flex           0.2
-   :flex-direction "column"
-   :align-items    :center
-   :padding-top    16})
+  {:flex-direction "column"})
+
+(def popular-list-chat-action
+  {:background-color "rgb(220, 214, 251)"
```

Comment:
Same color and column in this ns as before

Reply:
colors should be with other colors